README.md

```diff
@@ -15,13 +15,21 @@
 4. `cd` into the repo using Command Prompt/Terminal
 5. Run the following command:
      `vagrant up`
-5. This would be a good time to rewatch your favorite TV series installment, cook & have breakfast/lunch/dinner, walk the dog, clean your room, etc.<br>
-   By the time you get back the install will surely have completed.<sup>1</sup>
-6. ???
-7. Profit!
+6. Once vagrant exists with an error, execute `vagrant up` again (Workaround to install the linux headers required by the guest additions)
+7. This would be a good time to rewatch your favorite TV series installment, cook & have breakfast/lunch/dinner, walk the dog, clean your room, etc.<br>
+By the time you get back the install will surely have completed.<sup>1</sup>
+8. To confirm the installation worked, open the web browser of your choice and enter `http://192.168.64.78:3000/` into the address bar and see if the website loads correctly.
```

Comment:
This Vagrantfile actually specifies that you should connect to `http://e621.lc`. The IP address can change, but the hostname will not.

Reply:
Good point, must have missed that for some reason.
Thanks